### DISCUSSION TITLE
Streamer settings: Following Only IRC?

### DISCUSSION THREAD

**Discussion by 13Quavo** [RakambdaOrg/ChannelPointsMiner#256] [Q&A] [answered]
Hello,

Is it possible to only watch streamer if the account follows?

Thank you!

**Comment by Rakambda** [comment, marked as answer]
You mean watch a streamers only if that streamer is following your account? If so then no.

Actual answer here: https://github.com/RakSrinaNa/ChannelPointsMiner/discussions/256#discussioncomment-3485347

**Reply by 13Quavo** [reply]
I mean, only if my account follows that streamer

**Reply by Rakambda** [reply]
Yes, it's described in the readme about the configuration.
https://github.com/RakSrinaNa/ChannelPointsMiner#account-settings (loadFollows)
